docs/examples/DataSet/Offline Plotting Tutorial.ipynb

```diff
@@ -6,14 +6,14 @@
    "source": [
     "# Offline Plotting Tutorial\n",
     "\n",
-    "The dataset comes with a tool for offline (i.e. not live as the data are coming in) plotting. This notebook explains how to use it and what it is capable of plotting. **NOTE**: This notebook only covers the plotting of numerical data. For categorical (string-valued) data, please see `Offline plotting with categorical data.ipynb`.\n",
+    "The dataset comes with a tool for offline (i.e. not live as the data are coming in) plotting. This notebook explains how to use it and what it is capable of plotting. **NOTE**: This notebook only covers the plotting of numerical data. For categorical (string-valued) data, please see [Offline plotting with categorical data](http://qcodes.github.io/Qcodes/examples/DataSet/Offline%20plotting%20with%20categorical%20data.html).\n",
```

Comment:
please use relative path to ipynb files in the references, not github.io urls. THis is for consistency of the documentation and independences of its deployment

Reply:
E.g. this means that if you click on the link from a local copy of the notebook it will take you to the notebook locally running on your local notebook server ready to execute. See here for more info https://nbsphinx.readthedocs.io/en/0.4.3/markdown-cells.html#Links-to-Other-Notebooks on how this works.